contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -97,6 +103,11 @@ contract RaidenMicroTransferChannels {
         require(token.totalSupply() > 0);
 
         challenge_period = _challenge_period;
+        for (uint256 i = 0; i < _trusted_contracts.length; i++) {
```

Comment:
With a large enough `_trusted_contracts` array this loop can botch the contstructor and make it run out of gas. Is that okay? What is the average expected size of this array? How many trusted contracts will we be setting for a microraiden channel manager?

Reply:
Current plan is to have 1 trusted contract - ex. market maker through which a user can open a channel by sending ETH instead of the channel token.
Thought about the out of gas possibility, but at this point, even the trusted contracts are something that we control and develop. I see no reason to have more than 1-2 trusted contracts.
Should be discussed more.